tensorflow_encrypted/protocol/pond.py

```diff
@@ -216,8 +216,8 @@ def helper(v: tf.Tensor):
             assert v.shape.is_fully_defined(), "Shape of input '{}' on '{}' is not fully defined".format(
                 name if name else '', provider.player.name)
 
-            v = _encode(v, apply_scaling)
-            x0, x1 = _share(v)
+            var = _encode(v, apply_scaling)
```

Comment:
`v` was for "value" here, `var` could be misleading

Reply:
use `w` or `val` perhaps?

Reply:
Yep sure!